appveyor.yml

```diff
@@ -10,9 +10,9 @@ init:
 environment:
   matrix:
     # node.js
-    - nodejs_version: 4
```

Comment:
We should keep Node 4 for now I think.

Reply:
I think you may remove node4 support with 0.65 release

Reply:
Node 4 has been out of active and maintenance LTS since April, and has passed end of live and is unsupported. (https://github.com/nodejs/Release#end-of-life-releases) This is a prudent change.

Reply:
Fair enough. Given that I agree with @shellscape to remove it for `0.65`. But we should have a discussion before we actually change the TypeScript output target to match that in the future as this would have some hidden implications.

Reply:
Debian stable packages 4.8.2, and since sindresorhus/pretty-ms@b5a583de262d57a6a77b279f311ba9deefb2717d, rollup doesn't run on nodejs 4 anymore.